jenkins-integration/beaker/install/shared/stop_sut_background_scripts.rb

```diff
@@ -0,0 +1,12 @@
+require 'json'
+
+step "Stop background scripts on SUT" do
+  pids = JSON.parse(File.read("bg_pids.json"))
+
+  Beaker::Log.notify("Stopping #{pids.count} background scripts")
+  Beaker::Log.notify(JSON.pretty_generate(pids))
+  pids.each do |k, v|
+    Beaker::Log.notify("Stopping script '#{k}' with pid '#{v}'")
+    on(master, "kill #{v}")
```

Comment:
For our while looper script, hopefully it will just die fast enough it being alive to potentially keep writing stuff when the archiving kicks in later on won't be a problem.  I haven't practically run into this race condition yet, just seemed like it could happen.

I haven't run that bash line via beaker but I have run that bash line on one of the SUTs locally from the shell and it seemed to work well for looping until the process died.

If you want to just wait and see if we have a problem, I'm fine with that.